controllers/storageclassclaim_controller.go

```diff
@@ -406,7 +406,7 @@ func (r *StorageClassClaimReconciler) reconcilePhases() (reconcile.Result, error
 		}
 
 		// Delete configmap entry for cephcsi
-		err = csi.UpdateMonConfigMap(r.ctx, r.Client, r.log, r.storageClassClaim.Name, r.storageClient.Status.ConsumerID, nil)
+		err = csi.UpdateMonConfigMap(r.ctx, r.Client, r.log, r.OperatorNamespace, r.storageClassClaim.Name, r.storageClient.Status.ConsumerID, nil)
```

Comment:
When you're passing in this many variables that's usually a sign that the function needs to be refactored. I think you need to reconsider what goes in the CSI package at this point, you might want to have anything that requires a Client be moved to the reconciler.

Reply:
One reason for not moving this to reconcile is that this is a common function for both reconcile and status-reporter job. Moved common input to a new struct and made this as method to avoid more inputs